### PR TITLE
Storage set

### PR DESCRIPTION
**Pull request by adrianespejo**
StorageSet working with basic types and tuples

### CODE REVIEW

File: hecuba_py/src/hset.py [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+                query += str(value)
+            result = config.session.execute(query)
+            # result[0] is the first row (will be only on row) and result[0][0] is the count
+            return result[0][0]
+        else:
+            return set.__contains__(self, value)
+
+    def union(self, storageSet):
+        if not isinstance(storageSet, StorageSet):
+            raise Exception("Expected StorageSet argument")
```
Comment (cugni) [inline hecuba_py/src/hset.py]: the union of a storageset and a normal set should work

File: hecuba_py/src/hset.py [hunk 2]
```diff
@@ -0,0 +1,421 @@
+from IStorage import IStorage, AlreadyPersistentError
+from hecuba import config, log
+from collections import namedtuple
+import re
+import uuid
+
+class Iterator:
```
Comment (cugni) [inline hecuba_py/src/hset.py]: remove this class. Use a the function map instead to select what you need.
e.g.
```python
it = iter_of_something()
map(lambda a:a[0],it)
```

**Comment by polsm91** [comment]
What's the status? What did we agree on?

**Comment by adrianespejo** [comment]
> What's the status? What did we agree on?

This is an old version of the StorageSet implementation, the good one is already merged. We can delete this pull request.

**Pull request closed without merging**